ov/profiles.go

```diff
@@ -492,6 +492,7 @@ func (c *OVClient) CreateProfileFromTemplate(name string, template ServerProfile
 		new_template.Type = "ServerProfileV12"
 	}
 	new_template.ServerProfileTemplateURI = template.URI // create relationship
+	new_template.Description = template.ServerProfileDescription
```

Comment:
in line 501, its getting over written 
is that something we missed or we are deliberately doing this?
If so, then please add comments.